src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+
+    public static void printList(String[] tasks, int taskCount, int[] taskDone) {
+        String[] currList = Arrays.copyOf(tasks, taskCount + 1);
+        System.out.println("____________________________________________________________");
+        System.out.println("PENDING HIT LIST:");
+        for (int i = 0; i < currList.length; i++) {
+            if (currList[i] != null) {
+                if (taskDone[i] == 1) {
+                    System.out.println(i + 1 + "." + " [x] " + currList[i]);
+                }
+                else {
+                    System.out.println(i + 1 + "." + " [ ] " + currList[i]);
+                }
+            }
+        }
+        System.out.println("____________________________________________________________");
+
+    }
+
+    public static int doneTask(String line, String[] tasks) {
+        String[] number = line.split(" ");
```

Comment:
Can have a more descriptive variable name as number is quite a vague name

Reply:
Yes, I agree as at a glance the variable number may not be apparent to the reader/reviewer